twitter.css

```diff
@@ -0,0 +1,632 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.a{
+    border: 1px solid  rgb(203, 200, 200);
+}
+.b{
+    border-top:1px solid rgb(203, 200, 200);
+    border-bottom:1px solid rgb(203, 200, 200);
+
+}
+.hashtag-blue{
+    color: #00acee;
+}
+/* header  */
+.header{
+    display: grid;
+    grid-template-columns: minmax(260px,300px) minmax(400px,1fr) 300px;
+    background-color: #ffffff;
+    height:40px;
+}
+.logo{
+    font-size: 30px;
+    color: #00acee;
+    padding-left: 60px;
+}
+
+/* left-header */
+.left-header{
+    background-color: rgb(255, 255, 255);
+    padding-top:5px;
+}
+/* main-header */
+.main-header{
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    padding: 10px;
+    border-bottom: none;
+}
+.main-header > a:nth-child(1){
+    text-decoration: none;
+    color: black;
+    font-weight: bold;
+    transition: .3s;
+}
+.main-header > a:nth-child(1):hover{
+    color: #00acee;
+    transition: .3s;
+}
+.main-header > a:nth-child(2){
+    text-decoration: none;
+    display: grid;
+    justify-content: end;
+}
+/* right-header */
```

Comment:
I like how you've placed comments over the new sections of the page to make it clearer for yourself or other viewers to understand where you're working.